src/dotnet/UI.Blazor/Layouts/base-layout.ts

```diff
@@ -0,0 +1,6 @@
+import { onDeviceAwake } from '../../../nodejs/src/on-device-awake';
```

Comment:
you can import just as `from 'on-device-awake'`

Reply:
done